portafly/src/i18n/locales/en/strings.js

```diff
@@ -0,0 +1,48 @@
+export const OVERVIEW = {
+  page_title: 'Overview | Portafly',
+  page_title_description: 'Title of Overview page for tab browser',
```

Comment:
Every strings needs to have a description, to avoid false friends in translations